src/components/Modal/README.md

```diff
@@ -684,6 +752,7 @@ See Apple’s Human Interface Guidelines and API documentation about accessibili
 
 - Modals use ARIA `role=”dialog”` to convey to screen reader users that they work like native dialog windows.
 - If you set the `title` prop to give the modal component a heading, then the `title` is used to label the dialog element with `aria-labelledby`. This helps to convey the purpose of the modal to screen reader users when it displays.
+- After a modal is closed, in order to return focus to the button that launched it, pass the button to the modal as an `activator`.
```

Comment:
This sentence is very similar to the one in the `keyboard support` section,

> After a modal is closed, focus returns to the button that launched it

I feel like it makes the activator concept stand out if its explained in two places. Removing it from the accessibility section would lose the implementation details, and removing it from the keyboard support section might make it seem like we stopped supporting this feature. What do y'all think?